src/main/resources/api/swagger.yaml

```diff
@@ -222,6 +218,15 @@ definitions:
             $ref: '#/definitions/Link'
           next:
             $ref: '#/definitions/Link'
+  Review:
+    type: object
+    properties:
+      review-status:
```

Comment:
Because we have a property status in the features resource too. But the review-status != features.status (review-status cannot be pending). To not confuse those two, i chose the introduce the prefix.